indico_toolkit/indico_wrapper/indico_wrapper.py

```diff
@@ -57,13 +61,46 @@ def train_model(
             )
         )
 
-    def get_storage_object(self, storage_url):
+    def _retry(exceptions, wait=0.5):
```

Comment:
What's the rationale behind making this private?

Reply:
didn't want someone trying to call self.retry(), it doesn't really match the other methods in IndicoWrapper. But it might be useful to leave it exposed and docstring it

Reply:
shouldn't retry be a standalone function? why make it a class method?

Reply:
made it a class method because I thought other classes like workflow might need retry, or other classes that inherit

Reply:
they might, but they could just import that function (I think probably makes most sense as a stand alone function)